src/oas2/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { isObject } from 'lodash';
+import { Schema } from 'swagger-schema-official';
+
+export function getExamplesFromSchema(data?: Schema) {
+  let examples;
```

Comment:
Prefer a ternary operator to avoid the mutable reference.
```suggestion
  const examples = …
```